packages/@cdktf/hcl2json/build-go.sh

```diff
@@ -1,6 +1,12 @@
 #!/bin/bash
 set -euo pipefail
 
+# We use go.work for examples (so we can have editor support in those)
+# We have to disable it here however, because else this script would try to
+# build all go workspaces (which includes the examples which rely on the cdktf
+# go package which might not exist yet)
+export GOWORK=off
```

Comment:
This is a bit weird, but I'm ok with it for now

Reply:
It's because we generate Go code (`cdktf`, through JSII) that we use elsewhere in our project 😅
Using a `go.work` file however, gives us autocomplete in all the Go examples, which is reallyyyy helpful when writing them.